src/styles/globals.css

```diff
@@ -54,6 +54,18 @@ body {
     color: transparent;
   }
 
+  /* To style button/container without using :before and :after to have color transition */
+  .brand-bg-gradient-1 {
```

Comment:
do we still need this?

Reply:
Yes, this is what I added for the button in this PR

Reply:
what if we use `.brand-gradient-1`?

Reply:
wont work for this case, i need the first linear-gradient part to hack around and keep the black background:
`linear-gradient(#000000, #000000) padding-box,`